src/app/page.tsx

```diff
@@ -18,10 +22,10 @@ type Stat = {
 };
 
 const stats: Stat[] = [
-  { number: '180+', label: 'Pro Bono Cases' },
-  { number: '200+', label: 'Attorney Volunteers' },
-  { number: '65+', label: 'Interpreters & Translators' },
-  { number: '50+', label: 'Successful Outcomes' },
+  { number: '180+', label: 'Matched Cases' },
+  { number: '180+', label: 'Attorney' },
+  { number: '65+', label: 'Interpreters' },
+  { number: '45+', label: 'Successful Results' },
```

Comment:
can we revert back the stats section to match Ambreen's requested wording + stats: 
<img width="914" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/c742f618-dc59-46c4-b3cd-6c6372af79cf">